meson.build

```diff
@@ -231,7 +231,7 @@ endforeach
 compile_time_cpuflags = []
 
 # get binutils version for the workaround of Bug 97
-ldver = run_command('ld', '-v').stdout().strip()
+ldver = run_command('ld', '-v', check: true).stdout().strip()
```

Comment:
This is fragile, because CC_LD may be gold or lld or mold and thus binutils versions aren't relevant. That being said, `find_program()` has a version kwarg and you can use it to look for specific versions -- if it isn't found, then ld probably exists but is a different version.

I added a `.version()` method in 0.62.0 (I know, very new :() so you can do `ldver = find_program('ld').version()`.

Reply:
The newer .version() support would be great, but we are limited to older versions for now.

Reply:
Thank you for the review, if we can find a safe way to support the latest meson/ninja I would be happy to do it.